test/sqllogictest/aggregates.slt

```diff
@@ -398,3 +398,22 @@ FROM
     v
 ----
 {"0":"c","1":"a","2":"b"}  {"0":"c","1":"a","2":"d"}  {"0":"c","1":"a","2":"d"}  {"0":"c","1":"a","2":"b"}
+
+
+# Test Reduction elision
+
+statement ok
+CREATE TABLE a (x text)
+
+statement ok
+INSERT INTO a VALUES ('a'),('b')
+
+query T
+SELECT STRING_AGG(y, ', ') FROM (SELECT x as y FROM a);
+----
+a, b
+
+query T
+SELECT STRING_AGG(y, ', ') FROM (SELECT x as y FROM a limit 1);
```

Comment:
The aliasing to y both here and above don't seem to be necessary to repro the crash, can it be removed for simplification?

Reply:
the intent is to limit the input into the aggregate function. For example a single SELECT statement will not work as desired:
```
materialize=> SELECT STRING_AGG(x, ',') FROM a limit 1;
 string_agg 
------------
 a,b
(1 row)
```

I could get rid of the aliasing and use `select *` like this:
```
SELECT STRING_AGG(x, ', ') FROM (SELECT * FROM a ORDER BY x limit 1);
 string_agg 
------------
 a
(1 row)
```

What do you think?

Reply:
Sure, do whatever you need. Just wanted to make the test cases as minimal as possible.